### DISCUSSION TITLE
Suggest me a css learning site

### DISCUSSION THREAD

**Discussion by ghost** [community/community#83120] [Programming Help] [answered]
### Body

Hi I'm @Yuheshpandian I had learnt some basic concepts of html and strong on it. Now I want to learn css. I learnt html from a youtube tutorial of Dave gray, but his css vid is not for me. I want to learn css through websites, as i dont want to get trapped in tutorial hell. Help me to do so!.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by anuragjain-git** [comment]
There are several great websites but few popular ones are:

1. [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS)
2. [W3Schools CSS Tutorial](https://www.w3schools.com/css/)

**Reply by ghost** [reply]
thank you i will look out for these

**Comment by m-mourouh** [comment]
1. [W3Schools](https://www.w3schools.com/css/) 
2. [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS)

**Reply by ghost** [reply]
Yeah I looked onto them already thank you

**Comment by MaorBezalel** [comment, marked as answer]
Highly recommend [cssreference.io](https://cssreference.io/).

**Reply by ghost** [reply]
This website was actually fire, I don't know why its too underrated, but it has many useful definitions and code snippets for each simple but muster properties in css.

Thank You @MaorBezalel

**Reply by MaorBezalel** [reply]
@Yuheshpandian, you're most welcome!

Allow me to recommend another underrated platform, [Frontend Mentor](https://www.frontendmentor.io/), if you're eager to break free from tutorial hell. This platform offers practical challenges with design mockups, allowing you to apply HTML and CSS in real projects. Furthermore, it encourages deploying and hosting the solution projects you've built, providing an additional valuable learning tool.

I suggest starting with the introductory challenges that focus solely on HTML and CSS. As you grow more comfortable, progress to more demanding tasks involving JavaScript. Once you feel confident with HTML, CSS, and vanilla JavaScript, these challenges can serve as a gateway to explore various JavaScript frameworks and libraries such as React, Vue.js, etc.

**Reply by ghost** [reply]
Thank you for your recommendations, these ones are awesome you are so helpful to me.

**Comment by zanehirning** [comment]
I would recommend [The Odin Project](https://www.theodinproject.com/).

They have tons of tutorials on html, css, JavaScript, etc. 

There tutorials are very helpful and also provide some projects at the end of their "paths". Highly recommend.

**Reply by ghost** [reply]
Thank you for your great suggestions these sure would benefit me.

**Comment by jackl57** [comment]
I’m actually looking for the same thing. I learned HTML pretty easily, but CSS tutorials get overwhelming fast. I want to pick it up through practical, website-based resources because I’m trying to build a small tool similar to [allbratgenerator](https://allbratgenerator.com/), so I really need to understand CSS properly.

If you find any good sites for learning step-by-step, share them — I’m searching for the same kind of approach.
